docs/cli/projects/get-started.md

```diff
@@ -0,0 +1,255 @@
+---
+title: Get started
+sidebar_position: 2
+---
+
+In this tutorial, we'll explore how to get the IP address of the machine your code is running on and deploy your code to the RunPod platform.
+
+By the end, you'll have a solid understanding of how to set up a project environment, interact with your code, and deploy your code to a Serverless Endpoint on the RunPod platform.
+
+While this project is scopped to getting the IP address of the machine your code is running on, you can use the RunPod platform to deploy any code you want.
+For larger projects, bundling large packages to Docker image and making code changes requires multiple steps.
+With RunPod's development server, you can make changes to your code and test them in a live environment without having to rebuild a Docker image or redeploy your code to the RunPod platform.
```

Comment:
a RunPod development server? You've mentioned avoiding the possessive form of RunPod before